File: app/src/main/java/com/example/myapplication/models/animemodels/AnimeNewReleaseResultModel.java
package com.example.myapplication.models.animemodels;

import androidx.annotation.NonNull;

import java.io.Serializable;
import java.util.List;

import lombok.Data;

@Data
public class AnimeNewReleaseResultModel implements Serializable {

    private String animeEpisode;
    private String animeEpisodeNumber;
    private String animeEpisodeType;
    private String animeEpisodeStatus;
    private String episodeThumb;
    private String episodeURL;

    @NonNull
    @Override
    public String toString() {
        return "AnimeNewReleaseResult{" +
                "animeEpisode='" + animeEpisode + '\'' +
                ", animeEpisodeNumber='" + animeEpisodeNumber + '\'' +
                ", animeEpisodeType='" + animeEpisodeType + '\'' +
                ", animeEpisodeStatus='" + animeEpisodeStatus + '\'' +
                ", episodeThumb='" + episodeThumb + '\'' +
                ", episodeURL='" + episodeURL + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/myapplication/models/animemodels/AnimeNewReleaseResultModel.kt
package com.example.myapplication.models.animemodels

import java.io.Serializable

data class AnimeNewReleaseResultModel(
        var animeEpisode: String? = null,
        var animeEpisodeNumber: String? = null,
        var animeEpisodeType: String? = null,
        var animeEpisodeStatus: String? = null,
        var episodeThumb: String? = null,
        var episodeURL: String? = null
) : Serializable {
    override fun toString(): String {
        return "AnimeNewReleaseResult{" +
                "animeEpisode='" + this.animeEpisode + '\''.toString() +
                ", animeEpisodeNumber='" + this.animeEpisodeNumber + '\''.toString() +
                ", animeEpisodeType='" + this.animeEpisodeType + '\''.toString() +
                ", animeEpisodeStatus='" + this.animeEpisodeStatus + '\''.toString() +
                ", episodeThumb='" + this.episodeThumb + '\''.toString() +
                ", episodeURL='" + this.episodeURL + '\''.toString() +
                '}'.toString()
    }
}